gm4_blossoming_pots/overlay_48/data/gm4_blossoming_pots/function/decorated/as_player.mcfunction

```diff
@@ -0,0 +1,15 @@
+# Sets up raycast, then redirects to as_marker, then revokes advancement
+# @s = player who interacted with a decorated pot
+# at @s
+# run from advancement gm4_blossoming_pots:item_in_pot
+
+# Raycast for the pot (set up range and search)
+execute store result score @s gm4_blossoming_pots.range run attribute @s minecraft:player.block_interaction_range get 1
```

Comment:
You don't actually need these overlays. Our `gm4/plugins/backwards.py` script will automatically find any attributes and create overlays for them.

Reply:
Ah, awesome, noted. I'll have it implement that in a few hours when I get home.